File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.java
class Solution {
    public String solution(String s) {
        String answer = "";
        
        if(s.length() % 2 == 0){ //짝수
            answer = s.substring(s.length()/2-1, s.length()/2+1);
        }else { //홀수
            answer = s.substring(s.length()/2, s.length()/2+1);
        }

        return answer;
    }
}

File: 프로그래머스/1/12903. 가운데 글자 가져오기/가운데 글자 가져오기.js
function solution(s) {
    var answer = '';
    let mid = 0;
    
    if(s.length % 2 !== 0) {
        mid = Math.round(s.length / 2);
        answer = s.slice(mid - 1, mid);
    } else {
        mid = s.length / 2;
        answer = s.slice(mid - 1, mid + 1);
    }
    return answer;
}